src/pages/Appointments/Appointments.jsx

```diff
@@ -1,5 +1,31 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 
-const Appointment = () => <div>Appointment under construction</div>;
+import { fetchAp, fetchD } from '../../redux/thunk/api';
+
+import AppointmentCard from './AppointmentCard';
+
+const Appointment = () => {
+  const appointment = useSelector((state) => state.appointments);
+  const doctors = useSelector((state) => state.doctor);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchAp(2));
+    dispatch(fetchD());
+  }, [dispatch]);
```

Comment:
When I visited the appointments page, it's not showing any card with the information about the appointments, the console is showing the next error:
![Captura de Pantalla 2022-04-04 a la(s) 17 55 30](https://user-images.githubusercontent.com/88604987/161646051-3a35ec45-db3b-4d00-9bf3-8fb77b12c745.png)

And the API has a record that should be displayed!
![Captura de Pantalla 2022-04-04 a la(s) 17 59 34](https://user-images.githubusercontent.com/88604987/161646163-1a096300-eac5-4ece-9380-de30021e742a.png)



Kindly fix it!

Reply:
@AlexRS90 ,
Thank you for your review. Please note that the path to get an appointment is `http://localhost:3000/api/v1/:user_id/appointments` and at the moment there is no way to get the current user until the `user ui` is done with. Therefore, in the `src/pages/Appointments/Appointments.jsx` we need to `hard-code` `user_id`. This is the reason you are getting the error. Kindly pass in the `user_id` into the `fetchApp` method and the error will go away. Thank you.

Reply:
Hello, @micheaol yeah, you're right, it was my bad, I hard-coded the user-id and the app is fetching the data as expected, but now is throwing a warning! ⚠️ see the picture below ⬇️ 

![Captura de Pantalla 2022-04-04 a la(s) 22 20 12](https://user-images.githubusercontent.com/88604987/161672682-670a1de3-0f82-4b0a-9b6a-655ef1dde709.png)

Reply:
@AlexRS90 ,

Thank you for the review.
The key issue will be fixed.